### DISCUSSION TITLE
Multi Camera ambiguities warning and a dynamic-feature breakpoint issue

### DISCUSSION THREAD

**Discussion by Parallel-Paradox** [bevyengine/bevy#5749] [Q&A] [answered]
## Bevy version

0.8.0

## What you did

Create an Orthographic camera:
```rust
    commands.spawn_bundle(Camera3dBundle {
        transform: Transform::from_xyz(5.0, 2.0, 5.0).looking_at(Vec3::ZERO, Vec3::Y),
        projection: OrthographicProjection {
            scale: 3.0,
            scaling_mode: ScalingMode::FixedVertical(2.0),
            ..default()
        }.into(),
        ..default()
    });
```

And an ui camera:
```rust
commands.spawn_bundle(Camera2dBundle::default());
```

## What went wrong

```
2022-08-20T11:16:29.212152Z  WARN bevy_render::camera::camera_driver_node: 
Camera priority ambiguities detected for active cameras with the following priorities: 
{(0, Window(WindowId(00000000-0000-0000-0000-000000000000)))}. 
To fix this, ensure there is exactly one Camera entity spawned with a given priority for a given RenderTarget. 
Ambiguities should be resolved because either
 (1) multiple active cameras were spawned accidentally, which will result in rendering multiple instances of the scene or
 (2) for cases where multiple active cameras is intentional, ambiguities could result in unpredictable render results.
```

How to set the priority of multiple cameras?

# Additional Question

bevy_dylib_xxxx.dll and std_xxxx.dll cannot found in win11, when add breakpoint and debug in Clion.

I've run `rustc --print target-libdir` and set env like this:

![image](https://user-images.githubusercontent.com/81791934/185743846-29e1b7ed-b3fa-40c3-91d7-781c4a6a8b37.png)

when disable `--features bevy/dynamic`, the breakpoints goes well.

THX A LOT!

**Comment by DJMcNab** [comment, marked as answer]
```rust
commands.spawn_bundle(Camera2dBundle::default());
```
Isn't a UI camera. You no longer need an explicit UI camera, I think?

To set the priorities, you set the `priority` field on the `Camera` component.

**Comment by afonsolage** [comment]
About your additional question, I'm using VSCode on W11 and I don't use the `[..]\lib\rustlib[..]` path, instead I point to `[..]\bin` path. Here is my launch env var:
```json
"env": {
    "CARGO_MANIFEST_DIR": "${workspaceFolder}",
    "PATH": "${env:USERPROFILE}/.rustup/toolchains/nightly-x86_64-pc-windows-msvc/bin;${workspaceFolder}/target/debug/deps;${env:PATH}",
},
```
This works with `bevy/dynamic`

**Comment by hammackj** [comment]
You didn't post all the code but if its anything like that I had with a .add_system(setup) it was looking to add the camera. Changing it to .add_startup_system() fixed it for me.
